samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_accelerometer.c

```diff
@@ -259,7 +259,7 @@ int lwm2m_init_accel(void)
 				  &timestamp, sizeof(timestamp), 0);
 
 	if (IS_ENABLED(CONFIG_FLIP_POLL)) {
-		k_delayed_work_submit(&flip_poll_work, K_NO_WAIT);
+		k_work_schedule(&flip_poll_work, K_NO_WAIT);
```

Comment:
There are several like this where it looks like `k_work_reschedule()` might be appropriate, to cut short the delay if the work is already scheduled.  But maybe that can't happen.

Reply:
Here the module is initialized and the work is scheduled for the first time. Then it is scheduled again only from its own handler. That's why I used `k_work_schedule()` in both these cases.